libs/api/domains/documents/src/lib/document.service.ts

```diff
@@ -33,21 +41,26 @@ export class DocumentService {
     }
   }
 
-  async listDocuments(nationalId: string): Promise<Document[]> {
+  async listDocuments(
```

Comment:
We might want to publish these functions as version2 and keep the others in, while the app still uses them so we don't catch them by surprise when we change the responses. 😃